_data/prefixes.yaml

```diff
@@ -1,11 +1,13 @@
 covjson: https://covjson.org/def/core#
 covjsondt: https://covjson.org/def/domainTypes#
 xsd: http://www.w3.org/2001/XMLSchema#
-ssn: http://www.w3.org/2005/Incubator/ssn/ssnx/ssn#
+ssn: http://www.w3.org/ns/sosa#
```

Comment:
Would the prefix not be "sosa"? I thought in the ticket the proposal was to keep the `ssn` namespace and add a new one.